### PR TITLE
docs: added 60 new contributors details (#31992)

### PR DESCRIPTION
**Pull request by PandiyanCool**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by StephenFluin** [comment]
Can you make sure all of the images are <30kb and remove the addition of Zack Chapple?

**Comment by PandiyanCool** [comment]
Sure @StephenFluin

**Review by gabrielbussolo — APPROVED** [review]
I verified file by file after the improvement and all have ~30kb :+1:
Looks good for me. :+1:

**Comment by PandiyanCool** [comment]
Thanks @gabrielferreirapro

**Comment by petebacondarwin** [comment]
@PandiyanCool - it looks like some of these contributors have already been added. Can you rebase and resolve the conflicts?

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -806,6 +806,477 @@
     "bio": "Kwinten is a GDE for Angular with a passion for reactive coding. He works as a software engineer for Stackblitz. He shares his passion for Angular and coding through blogposts and training sessions with the company he co-founded called Strongbrew.",
     "groups": ["GDE"]
   },
+  "abraham": {
```
Comment (mgechev) [inline aio/content/marketing/contributors.json:809]: Abraham is now a Googler, so we should remove him from this section.

**Comment by mgechev** [comment]
@PandiyanCool thank you for keeping the contributors list up to date! Would you be able to rebase? It could be a little slow merging changes in `contributors.json` because of the many folks changing the file.

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (mgechev) [reply aio/content/marketing/contributors.json:809]: There are a few other changes as well. I'd encourage you to go through the list on https://developers.google.com/community/experts/directory

**Comment by petebacondarwin** [comment]
Closing since this PR is now very stale and there has been response.
@PandiyanCool if you would like to help update the contributors still, please could you create a new PR? Thanks.

**Pull request closed without merging**